idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/db/VisitDBService.java

```diff
@@ -147,8 +161,22 @@ public Visit processEntity(String uuid, String docAction) throws Exception {
 					newPayment.saveEx();
 				}
 				visit.setPayments(paymentDBService.getPaymentsByOrderId(visit.getId()));
-			}
+			} 
 			return visit;
+		} else if (docAction.equalsIgnoreCase(DocAction.ACTION_Complete)) {
```

Comment:
What are these changes for?

Reply:
And since it's not reversal it will pass without processing

Reply:
Gotcha. Then this can move to line 164 since we want it executing in the block above.